tests/test_ios_testflight_notes.py

```
@@ -0,0 +1,112 @@
+
+
+def _commit(repo, path, subject):
+    full = os.path.join(repo, path)
+    os.makedirs(os.path.dirname(full), exist_ok=True)
+    with open(full, "a", encoding="utf-8") as f:
+        f.write(subject + "\n")
+    _git(repo, "add", "-A")
+    _git(repo, "commit", "-q", "-m", subject)
+
+
+def _gen(repo, base, audience="internal"):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_gen`

(ANN202)